modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/EffectsFragment.kt

```diff
@@ -272,26 +271,27 @@ class EffectsFragment : BaseDialogFragment(), CompoundButton.OnCheckedChangeList
         playbackManager.trackPlaybackEffectsEvent(event, props, SourceView.PLAYER_PLAYBACK_EFFECTS)
     }
 
-    @OptIn(ExperimentalCoroutinesApi::class)
     private fun FragmentEffectsBinding.setupEffectsSettingsSegmentedTabBar() {
         effectsSettingsSegmentedTabBar.setContent {
-            val podcast by viewModel.effectsLive.asFlow()
-                .mapLatest { it.podcast }
+            val podcastEffectsData by viewModel.effectsLive.asFlow()
+                .distinctUntilChangedBy { it.podcast }
```

Comment:
I think making it distinct by a podcast UUID should work as well? If yes it's less work to compare a string vs comparing a full podcast. But I'm not sure. Maybe you need to compare both UUID and playback effects.

Reply:
Compared by both UUID and playback effects in 228a172.